lyricsgenius/genius.py

```diff
@@ -307,9 +307,10 @@ def search_album(self, name=None, artist="",
                 print('Searching for "{s}"...'.format(s=name))
 
         if album_id:
+            search_term = "id:{id}".format(id=album_id)
             album_info = self.album(album_id, text_format).get('album')
         else:
-            search_term = "{s} {a}".format(s=name, a=artist).strip()
+            search_term = "'{s} {a}'".format(s=name, a=artist).strip()
```

Comment:
I don't know how Genius will react to single quotes in the search term and they might affect the search results. Why change them anyway? The print statement below is working fine.

Reply:
Sorry, I didn't notice the search term was also used in the line below :man_facepalming: 
Fixing it in the next commit.